csp.go

```diff
@@ -40,6 +40,10 @@ func ParsePolicy(policy string) (Policy, error) {
 				return Policy{}, err
 			}
 
+		case "block-all-mixed-content":
+		case "upgrade-insecure-requests":
```

Comment:
Switch statements in go don't fall through

Should combine them into one case instead like this:

```
case "block...", "upgrade...":
```
It would also be good to add a check that the argument length is 0 similar to how the other parsers do it it